web/ideabag2-web/src/views/IdeaDetail.vue

```diff
@@ -232,13 +265,22 @@ export default {
 			var date = new Date(milliseconds);
```

Comment:
I wrote this, I think. Please make it `return new Date(milliseconds),toLocaleDateString();`